lib/Elastica/Bulk/Action/AbstractDocument.php

```diff
@@ -100,7 +96,7 @@ public function getScript()
             return $this->_data;
         }
 
-        return null;
+        return;
```

Comment:
I assume this is done automatically by the cleanup script. I'm not a fan of "empty" return, but if we have an automated way to cleanup things, I can also "life" with this version.